### PR TITLE
Update Factsheets Overview #801

### LINKED ISSUE #801: Update text on factsheet page (Overview)
I suggest to update the text on the factsheet overview page to make it easier to understand and have a more neutral language (example to adjust: _to avoid repeated and long explications of the used tools as well as senseless printing_).

I can draft a new text for this. 
But before doing this it would be good to have answers to the following questions:

1. will the study factsheets (alpha) make the scenario factsheet element in the drop-down obsolete, or will both continue to exist?
2. If both continue to exist, how do they relate to each other, i.e. what do the scenario factsheets cover that is not included in the study factsheets (alpha)?
3. can we add, on each individual factsheet page, a very short description what these kind of factsheets are about? I think this would be helpful for users. If yes, I can draft texts for these short introductions too (would open separate tickets). 

cc @stap-m, @l-emele

### PR DESCRIPTION
**Pull request by Ludee**
Closes #801

### CODE REVIEW

**Review by jh-RLI — APPROVED** [review]
LGTM

File: modelview/templates/modelview/overview.html [hunk 1]
```diff
@@ -4,71 +4,57 @@
...
+    In the future, these scenario factsheets will also be succeeded by <a
+        href="https://openenergy-platform.org/factsheets/rdf/study/">Study
+    Factsheets (alpha)</a> that will hold more functionality.
+    It is based on RDF and the <a
+        href="https://github.com/OpenEnergyPlatform/oekg">Open Energy Knowledge
+    Graph (OEKG)</a> under development. Please note that this is just a preview
+    and not a fully functional feature yet.
+</p>
+
+<p>These study factsheets will help you also learn more about scenario contexts
```
Comment (han-f) [inline modelview/templates/modelview/overview.html:40]: Suggested change:
These study factsheets will inform about scenario contexts
    and enable you to explore scenario bundles. Scenario bundles tie together various
    information about a scenario available on the OEP. This
    includes information on the scenario projection data available, the frameworks and
    models applied, and the study context. The study factsheets will make
    these relations visible and usable.
    And not only that: the study factsheets will be based on the concepts
    available in the Open Energy Ontology (OEO).

File: modelview/templates/modelview/overview.html [hunk 2]
```diff
@@ -4,71 +4,57 @@
...
+    factsheets are a helpful place to visit.
+</p>
+<p>
+    Specifically, our current Scenario Factsheets provide some dedicated
+    information about specific scenarios for which projection data is available
+    on the OEP. Please note, however, that their format was
+    developed some time ago and they will undergo substantial updates.
+    In the future, these scenario factsheets will also be succeeded by <a
+        href="https://openenergy-platform.org/factsheets/rdf/study/">Study
+    Factsheets (alpha)</a> that will hold more functionality.
```
Comment (han-f) [inline modelview/templates/modelview/overview.html:33]: Suggested change:

These factsheets will be based on RDF and the <a
        href="https://github.com/OpenEnergyPlatform/oekg">Open Energy Knowledge
    Graph (OEKG)</a> which is currently under heavy development. Please note that this is just a preview
    and not a fully functional feature yet.

**Review by han-f — COMMENTED** [review]
I added two comments with small suggested changes to some parts.

**Comment by han-f** [comment]
thank you for putting in addition, too.

**Comment by Ludee** [comment]
Please also approve the PR if you are fine with the current state @han-f

**Comment by han-f** [comment]
I wanted to @Ludee , but didnt find the button to approve after having left the review area...
how can I do it?
I also cannot see the review button any longer

**Comment by jh-RLI** [comment]
@han-f you have to select the tab "Files changed" (below the PR title) and then there should be the "Review Changes" button in the top right corner. Then you can select the radio button for "approve" within the commend box.

**Review by han-f — APPROVED** [review]

**Comment by han-f** [comment]
thanks @jh-RLI - done!

**Pull request merged**